Dockerfile.mysql

```diff
@@ -0,0 +1,13 @@
+FROM mysql:latest
```

Comment:
Using `mysql:latest` is not recommended for production environments due to potential compatibility issues. Please specify a specific version tag to ensure stability.